packages/client/hmi-client/src/workflow/ops/simulate-ciemss/tera-simulate-ciemss.vue

```diff
@@ -156,6 +172,23 @@ const emit = defineEmits(['append-output', 'update-state', 'select-output', 'clo
 const inferredParameters = computed(() => props.node.inputs[1].value);
 
 const timespan = ref<TimeSpan>(props.node.state.currentTimespan);
+
+// Computed property to generate the example string
+const exampleString = computed(() => {
+	const { start, end } = timespan.value;
+	const step = (end - start) / (numSamples.value - 1);
+	const values = [];
+	for (let i = 0; i < numSamples.value; i++) {
+		const value = start + i * step;
+		if (value % 1 === 0) {
+			values.push(Math.floor(value));
+		} else {
+			values.push(value.toFixed(2));
+		}
+	}
+	return values.join(', ');
+});
+
```

Comment:
Number of samples is how many times you want to run the simulation, it has nothing to do with step-size. This exampleString is not necessary in CIEMSS, stepsize is always 1.

Reply:
Sorry, misunderstanding on my part based on some old designs. Removed.